passes/cmds/setundef.cc

```diff
@@ -137,7 +137,7 @@ struct SetundefPass : public Pass {
 		log("        replace with $anyconst drivers (for formal)\n");
 		log("\n");
 		log("    -random <seed>\n");
-		log("        replace with random bits using the specified integer als seed\n");
+		log("        replace with random bits using the specified integer seed\n");
```

Comment:
Not sure what "als" means here, I assumed a copy-paste error.

Reply:
Oops, yes. Should have been "as".  :)